iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/DeviceEmulator.java

```diff
@@ -54,26 +57,9 @@
         this.client = client;
     }
 
-    void setup() throws IOException
+    void open() throws IOException
     {
-        try
-        {
-            if (this.client != null)
-            {
-                this.client.closeNow();
-            }
-        }
-        catch (IOException e)
-        {
-            e.printStackTrace();
-        }
-
-        clearStatistics();
```

Comment:
Each test instance has its own statistics, so there is no need to clear them between test runs